packages/web-components/src/components/global-banner/global-banner.ts

```diff
@@ -140,7 +142,8 @@ class C4DGlobalBanner extends StableSelectorMixin(LitElement) {
     return html`
       <a
         href="${this.buttonHref}"
-        class="${prefix}--global-banner-layout-container">
+        class="${prefix}--global-banner-layout-container"
+        part="banner-button">
```

Comment:
Same situation here.
```suggestion
        part="container container--link">
```